src/non-flake-wrapper.nix

```diff
@@ -0,0 +1,14 @@
+{ mkShell }:
```

Comment:
1. Our flake exports a `mkShell` like this.
2. I don't particularly like the renamed arguments: `nixpkgs` and `configuration`. I would stick with the API we already expose (see 1.).